packages/venia-ui/lib/components/CheckoutPage/checkoutPage.js

```diff
@@ -162,6 +162,29 @@ const CheckoutPage = props => {
                 </h3>
             );
 
+        const isPaymentAvailable = !!availablePaymentMethods
+            .map(({ code }) => {
+                // If we don't have an implementation for a method type, ignore it.
+                if (!Object.keys(payments).includes(code)) {
+                    return;
+                }
+                return true;
+            })
+            .filter(paymentMethod => !!paymentMethod).length;
+
+        const errors = new Map();
+
+        if (!isPaymentAvailable) {
+            errors.set('paymentMethods', [
```

Comment:
I am not sure this needs to be a Map. This can be an Array and that opens up doors for future use cases as well. Other devs can just keep adding to the array instead of adding to the map and then modifying the `<FormError />` usage.